### ISSUE TITLE
PA Review: New account creation, sign-in, guest/demo sign-in

### ISSUE THREAD

**Issue by afarahmand** [afarahmand/kicks#1]
- [x] Backend: DB, model, controller, views
- [x]  Redux Loop: ajax, actions, reducer
- [x]  Presentational Components
- [x]  Styling
- [x]  Smooth, bug-free navigation
- [x]  Adequate and appropriate seeds

**Comment by AAlfarho** [comment]
Update the wiki to reflect the current link to the live app. Assign this issue back to me once the latest version of the app is pushed and the link is in the wiki.

**Comment by afarahmand** [comment]
Done!

**Comment by AAlfarho** [comment]
Good job on the user auth, the hovering style on the signin/signup form looks great.
There only a couple of details before we can mark this MVP as completed.

## Functionality
- [ ] As of now we are missing a default `route` on our `<Switch>`, create an issue to keep track of this, we will close that issue whenever we have the component we want to set as default ready.
- [ ] We are validating for the _users name_ length to be min 1 , is this validation really necessary? It seems that the `presence: true` would already take care of this. Consider removing that validation or increasing the min length.

Re-assign the issue back to me when the necessary updates are pushed to Heroku.

**Comment by afarahmand** [comment]
I went ahead and created a new issue for the default route.

**Comment by afarahmand** [comment]
I think our default route is ready and working.

**Comment by AAlfarho** [comment]
Great work on that landing page!

This MVP is now complete.
